src/AppInstallerCLICore/Command.cpp

```diff
@@ -868,9 +868,21 @@ namespace AppInstaller::CLI
         {
             std::string message = GetUserPresentableMessage(hre);
             Logging::Telemetry().LogException(command->FullName(), "winrt::hresult_error", message);
-            context.Reporter.Error() <<
-                Resource::String::UnexpectedErrorExecutingCommand << ' ' << std::endl <<
-                message << std::endl;
+            try

```

Comment:
I wonder if it would be better to have `ResolveString` swallow the exceptions and return the name string instead.  Obviously the experience would be poor, but then at least it would work.  The problem would be the legal wording that we are required to show wouldn't be present, so maybe it isn't actually an option.  But at the same time, this isn't really a supported scenario.

```
    std::string Loader::ResolveString(
        std::wstring_view resKey) const
    {
        try { return Utility::ConvertToUTF8(m_wingetLoader.GetString(resKey)); }
        CATCH_LOG();
        return Utility::ConvertToUTF8(resKey);
    }
```

Reply:
And now I've realized that the error comes not from `ResolveString` but from the constructor of the static.  So we could still do what I've said if we wanted, but not exactly how I've said it.